Script started on Tue 04 Apr 2023 10:54:44 AM CEST
bash: __git_ps1: command not found
]0;dock@reds-HP-Laptop: ~/workspace/src/gazebo_pkg_tests/slam_example/src[01;91mdock@reds-HP-Laptop[00m [01;94msrc[01;33m [00m $ zsh
[1m[7m%[27m[1m[0m                                                                                                                                                            ]2;dock@reds-HP-Laptop:~/workspace/src/gazebo_pkg_tests/slam_example/src]1;..m_example/src[0m[27m[24m[J[0m[49m[39m[0m[49m[38;5;208mÓÇ∂[0m[48;5;208m[37m Ôåõ [0m[48;5;208m[37m [0m[40m[38;5;208mÓÇ∞[0m[40m[33m dock[0m[40m[33m [0m[44m[30mÓÇ∞[0m[44m[30m ÔÅº  src[0m[44m[30m [0m[48;5;28m[34mÓÇ∞[0m[48;5;28m[38;5;220m Óúº  2.7.12 (py27)[0m[48;5;28m[38;5;220m [0m[43m[38;5;28mÓÇ∞[0m[43m[30m ÔÑì  [0m[43m[30mÔÑ¶ main[0m[43m[30m[0m[43m[30m ÔÅ™ [0m[43m[30m ÔÅô [0m[43m[30m [0m[49m[33mÓÇ∞[0m[49m[39m [0m[49m[39m[K[?1h=[?2004h[32ml[39m[32ml[39m[90ms[39m[32ml[32ms[39m[?1l>[?2004l
]2;lsd --group-dirs=first -S]1;ls[38;5;4m[1mÔÑï launch[0m  [38;5;4m[1mÔÑï scripts[0m  ÔÄñ typescript
[1m[7m%[27m[1m[0m                                                                                                                                                            ]2;dock@reds-HP-Laptop:~/workspace/src/gazebo_pkg_tests/slam_example/src]1;..m_example/src[0m[27m[24m[J[0m[49m[39m[0m[49m[38;5;208mÓÇ∂[0m[48;5;208m[37m Ôåõ [0m[48;5;208m[37m [0m[40m[38;5;208mÓÇ∞[0m[40m[33m dock[0m[40m[33m [0m[44m[30mÓÇ∞[0m[44m[30m ÔÅº  src[0m[44m[30m [0m[48;5;28m[34mÓÇ∞[0m[48;5;28m[38;5;220m Óúº  2.7.12 (py27)[0m[48;5;28m[38;5;220m [0m[43m[38;5;28mÓÇ∞[0m[43m[30m ÔÑì  [0m[43m[30mÔÑ¶ main[0m[43m[30m[0m[43m[30m ÔÅ™ [0m[43m[30m ÔÅô [0m[43m[30m [0m[49m[33mÓÇ∞[0m[49m[39m [0m[49m[39m[K[?1h=[?2004h[32mc[39m[32mc[39m[90md cmake[39m[32mc[32md[39m[39m [39m [39m [39m [39m [39m [90m cmake[39m[39m [39m [39m [39m [39m [39m [90mcmake[39m[39ms[39m [39m [39m [39m [90mervice[39m[39mc[39m [39m [39m [39m [39m r[4m[35ms[4m[35mc[4m[35mr[4m[35mipts/[0m[24m[39m[4m[35ms[24m[39m[24m[39m [?1l>[?2004l
]2;cd scripts]1;cddepth_to_laser.py  example.py  [0m[01;32mlaser_pointcloud.py[0m  [01;32mposeOdometry.py[0m  [01;32mscanLaser.py[0m  [01;32mslam_node.py[0m
[1m[7m%[27m[1m[0m                                                                                                                                                            ]2;dock@reds-HP-Laptop:~/workspace/src/gazebo_pkg_tests/slam_example/src/scripts]1;..e/src/scripts[0m[27m[24m[J[0m[49m[39m[0m[49m[38;5;208mÓÇ∂[0m[48;5;208m[37m Ôåõ [0m[48;5;208m[37m [0m[40m[38;5;208mÓÇ∞[0m[40m[33m dock[0m[40m[33m [0m[44m[30mÓÇ∞[0m[44m[30m ÔÅº  scripts[0m[44m[30m [0m[48;5;28m[34mÓÇ∞[0m[48;5;28m[38;5;220m Óúº  2.7.12 (py27)[0m[48;5;28m[38;5;220m [0m[43m[38;5;28mÓÇ∞[0m[43m[30m ÔÑì  [0m[43m[30mÔÑ¶ main[0m[43m[30m[0m[43m[30m ÔÅ™ [0m[43m[30m ÔÅô [0m[43m[30m [0m[49m[33mÓÇ∞[0m[49m[39m [0m[49m[39m[K[?1h=[?2004h[32mc[39m[32mc[39m[90md scripts[39m[9D[1m[31mc[1m[31mh[0m[39m[39m [39m [39m [39m [39m [39m [39m [39m [8D[90mmod +x laser_pointcloud.py[39m[26D[0m[32mc[0m[32mh[32mm[39m[39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [25D[90mod +x laser_pointcloud.py[39m[25D[1m[31mc[1m[31mh[1m[31mm[1m[31mo[0m[39m[39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [24D[90md +x laser_pointcloud.py[39m[24D[0m[32mc[0m[32mh[0m[32mm[0m[32mo[32md[39m[39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [23D[90m +x laser_pointcloud.py[39m[23D[1m[31mc[1m[31mh[1m[31mm[1m[31mo[1m[31md[1m[31me[0m[39m[39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [22D[0m[32mc[0m[32mh[0m[32mm[0m[32mo[0m[32md[39m[0m[39m [90m +x laser_pointcloud.py[39m[23D[39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [22D[90m+x laser_pointcloud.py[39m[22D[39m+[39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [21D[90mx laser_pointcloud.py[39m[21D[39mx[39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [20D[90m laser_pointcloud.py[39m[20D[39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [19D[90mlaser_pointcloud.py[39m[19D[39md[39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [18Depth_to_laser.p[16D[35md[35me[35mp[35mt[35mh[35m_[35mt[35mo[35m_[35ml[35ma[35ms[35me[35mr[35m.[35mp[35my[39m[?1l>[?2004l
]2;chmod +x depth_to_laser.py]1;chmod[1m[7m%[27m[1m[0m                                                                                                                                                            ]2;dock@reds-HP-Laptop:~/workspace/src/gazebo_pkg_tests/slam_example/src/scripts]1;..e/src/scripts[0m[27m[24m[J[0m[49m[39m[0m[49m[38;5;208mÓÇ∂[0m[48;5;208m[37m Ôåõ [0m[48;5;208m[37m [0m[40m[38;5;208mÓÇ∞[0m[40m[33m dock[0m[40m[33m [0m[44m[30mÓÇ∞[0m[44m[30m ÔÅº  scripts[0m[44m[30m [0m[48;5;28m[34mÓÇ∞[0m[48;5;28m[38;5;220m Óúº  2.7.12 (py27)[0m[48;5;28m[38;5;220m [0m[43m[38;5;28mÓÇ∞[0m[43m[30m ÔÑì  [0m[43m[30mÔÑ¶ main[0m[43m[30m[0m[43m[30m ÔÅ™ [0m[43m[30m ÔÅô [0m[43m[30m [0m[49m[33mÓÇ∞[0m[49m[39m [0m[49m[39m[K[?1h=[?2004h[32ml[39m[32ml[39m[90ms[39m[32ml[32ms[39m[?1l>[?2004l
]2;lsd --group-dirs=first -S]1;lsÓòÜ example.py  [38;5;2m[1mÓòÜ laser_pointcloud.py[0m  [38;5;2m[1mÓòÜ slam_node.py[0m  [38;5;2m[1mÓòÜ poseOdometry.py[0m  [38;5;2m[1mÓòÜ scanLaser.py[0m  [38;5;2m[1mÓòÜ depth_to_laser.py[0m
[1m[7m%[27m[1m[0m                                                                                                                                                            ]2;dock@reds-HP-Laptop:~/workspace/src/gazebo_pkg_tests/slam_example/src/scripts]1;..e/src/scripts[0m[27m[24m[J[0m[49m[39m[0m[49m[38;5;208mÓÇ∂[0m[48;5;208m[37m Ôåõ [0m[48;5;208m[37m [0m[40m[38;5;208mÓÇ∞[0m[40m[33m dock[0m[40m[33m [0m[44m[30mÓÇ∞[0m[44m[30m ÔÅº  scripts[0m[44m[30m [0m[48;5;28m[34mÓÇ∞[0m[48;5;28m[38;5;220m Óúº  2.7.12 (py27)[0m[48;5;28m[38;5;220m [0m[43m[38;5;28mÓÇ∞[0m[43m[30m ÔÑì  [0m[43m[30mÔÑ¶ main[0m[43m[30m[0m[43m[30m ÔÅ™ [0m[43m[30m ÔÅô [0m[43m[30m [0m[49m[33mÓÇ∞[0m[49m[39m [0m[49m[39m[K[?1h=[?2004h[32mr[39m[32mr[39m[90mostopic echo /scan[39m[18D[1m[31mr[1m[31mo[0m[39m[39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [17D[90mstopic echo /scan[39m[17D[1m[31mr[1m[31mo[1m[31ms[0m[39m[39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [16D[90mtopic echo /scan[39m[16D[1m[31ms[1m[31mr[0m[39m[39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [15D[90mun rviz rviz [39m[13D[1m[31mr[1m[31mu[0m[39m[39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [12D[90mn rviz rviz [39m[12D[0m[32mr[0m[32mo[0m[32ms[0m[32mr[0m[32mu[32mn[39m[39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [11D[90m rviz rviz [39m[11D[39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [10D[90mrviz rviz [39m[10D[39ms[39m [39m [39m [39m [39m [39m [39m [39m [39m [9D[90mlam_example laser_pointcloud.py[39m[31D[39ml[39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [30D[90mam_example laser_pointcloud.py[39m[30D[39ma[39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [29D[90mm_example laser_pointcloud.py[39m[29D[39mm[39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [28D[90m_example laser_pointcloud.py[39m[28D[39m_[39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [27D[90mexample laser_pointcloud.py[39m[27D[39me[39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [26D[90mxample laser_pointcloud.py[39m[26D[39mx[39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [25D[90mample laser_pointcloud.py[39m[25D[39ma[39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [24D[90mmple laser_pointcloud.py[39m[24D[39mm[39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [23D[90mple laser_pointcloud.py[39m[23D[39mp[39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [22D[90mle laser_pointcloud.py[39m[22D[39me[39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [21D depth_to_[28D[39mr[39mo[39ms[39mr[39mu[39mn[22C[28D[32mr[32mo[32ms[32mr[32mu[32mn[39m[22Claser.p[16D[35md[35me[35mp[35mt[35mh[35m_[35mt[35mo[35m_[35ml[35ma[35ms[35me[35mr[35m.[35mp[35my[39m[?1l>[?2004l
]2;rosrun slam_exampe depth_to_laser.py]1;rosrun[rospack] Error: package 'slam_exampe' not found
[1m[7m%[27m[1m[0m                                                                                                                                                            ]2;dock@reds-HP-Laptop:~/workspace/src/gazebo_pkg_tests/slam_example/src/scripts]1;..e/src/scripts[0m[27m[24m[J[0m[49m[39m[0m[49m[38;5;208mÓÇ∂[0m[48;5;208m[37m Ôåõ [0m[48;5;208m[37m [0m[40m[38;5;208mÓÇ∞[0m[40m[33m dock[0m[40m[33m [0m[44m[30mÓÇ∞[0m[44m[30m ÔÅº  scripts[0m[44m[30m [0m[48;5;28m[34mÓÇ∞[0m[48;5;28m[38;5;220m Óúº  2.7.12 (py27)[0m[48;5;28m[38;5;220m [0m[43m[38;5;28mÓÇ∞[0m[43m[30m ÔÑì  [0m[43m[30mÔÑ¶ main[0m[43m[30m[0m[43m[30m ÔÅ™ [0m[43m[30m ÔÅô [0m[43m[30m [0m[40m[33mÓÇ∞[0m[40m[31m ÔÄç[0m[40m[31m [0m[49m[30mÓÇ∞[0m[49m[39m [0m[49m[39m[K[?1h=[?2004h[32mrosrun[39m slam_exampe [35mdepth_to_laser.py[39mle[39m [35md[35me[35mp[35mt[35mh[35m_[35mt[35mo[35m_[35ml[35ma[35ms[35me[35mr[35m.[35mp[35my[39m[19D[?1l>[?2004l
]2;rosrun slam_example depth_to_laser.py]1;rosrunTraceback (most recent call last):
  File "/home/dock/catkin_ws/src/src/gazebo_pkg_tests/slam_example/src/scripts/depth_to_laser.py", line 5, in <module>
    from depth_to_laser import DepthImageToLaserScan
  File "/home/dock/workspace/src/gazebo_pkg_tests/slam_example/src/scripts/depth_to_laser.py", line 5, in <module>
    from depth_to_laser import DepthImageToLaserScan
ImportError: cannot import name DepthImageToLaserScan
[1m[7m%[27m[1m[0m                                                                                                                                                            ]2;dock@reds-HP-Laptop:~/workspace/src/gazebo_pkg_tests/slam_example/src/scripts]1;..e/src/scripts[0m[27m[24m[J[0m[49m[39m[0m[49m[38;5;208mÓÇ∂[0m[48;5;208m[37m Ôåõ [0m[48;5;208m[37m [0m[40m[38;5;208mÓÇ∞[0m[40m[33m dock[0m[40m[33m [0m[44m[30mÓÇ∞[0m[44m[30m ÔÅº  scripts[0m[44m[30m [0m[48;5;28m[34mÓÇ∞[0m[48;5;28m[38;5;220m Óúº  2.7.12 (py27)[0m[48;5;28m[38;5;220m [0m[43m[38;5;28mÓÇ∞[0m[43m[30m ÔÑì  [0m[43m[30mÔÑ¶ main[0m[43m[30m[0m[43m[30m ÔÅ™ [0m[43m[30m ÔÅô [0m[43m[30m [0m[40m[33mÓÇ∞[0m[40m[31m ÔÄç[0m[40m[31m [0m[49m[30mÓÇ∞[0m[49m[39m [0m[49m[39m[K[?1h=[?2004h[32mr[39m[32mr[39m[90mosrun slam_example depth_to_laser.py[39m[36D[1m[31mr[1m[31mo[0m[39m[39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [35D[90msrun slam_example depth_to_laser.py[39m[35D[1m[31mr[1m[31mo[1m[31ms[0m[39m[39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [34D[90mrun slam_example depth_to_laser.py[39m[34D[0m[32mr[0m[32mo[0m[32ms[32md[39m[39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [33D[90m rqt_robot_steering rqt_robot_steering[39m[38D[1m[31mr[1m[31mo[1m[31ms[1m[31md[1m[31me[0m[39m[39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [37D[90mp install slam_example[39m[22D[0m[32mr[0m[32mo[0m[32ms[0m[32md[0m[32me[32mp[39m[39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [21D[90m install slam_example[39m[21D[39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [20D[90minstall slam_example[39m[20D[39mi[39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [19D[90mnstall slam_example[39m[19D[39mn[39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [18D[90mstall slam_example[39m[18D[39ms[39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [17D[90mtall slam_example[39m[17D[39mt[39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [16D[90mall slam_example[39m[16D[39ma[39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [15D[90mll slam_example[39m[15D[39ml[39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [14D[90ml slam_example[39m[14D[39ml[39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [13D[90m slam_example[39m[13D[39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [12D[90mslam_example[39m[12D[39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [39m [12Dslam_example[?1l>[?2004l
]2;rosdep install slam_example]1;rosdep#All required rosdeps installed successfully
[1m[7m%[27m[1m[0m                                                                                                                                                            ]2;dock@reds-HP-Laptop:~/workspace/src/gazebo_pkg_tests/slam_example/src/scripts]1;..e/src/scripts[0m[27m[24m[J[0m[49m[39m[0m[49m[38;5;208mÓÇ∂[0m[48;5;208m[37m Ôåõ [0m[48;5;208m[37m [0m[40m[38;5;208mÓÇ∞[0m[40m[33m dock[0m[40m[33m [0m[44m[30mÓÇ∞[0m[44m[30m ÔÅº  scripts[0m[44m[30m [0m[48;5;28m[34mÓÇ∞[0m[48;5;28m[38;5;220m Óúº  2.7.12 (py27)[0m[48;5;28m[38;5;220m [0m[43m[38;5;28mÓÇ∞[0m[43m[30m ÔÑì  [0m[43m[30mÔÑ¶ main[0m[43m[30m[0m[43m[30m ÔÅ™ [0m[43m[30m ÔÅô [0m[43m[30m [0m[49m[33mÓÇ∞[0m[49m[39m [0m[49m[39m[K[?1h=[?2004h[32mrosdep[39m install slam_example[24D[32mr[32mu[32mn[39m[8P[13C [35mdepth_to_laser.py[39m[?1l>[?2004l
]2;rosrun slam_example depth_to_laser.py]1;rosrunTraceback (most recent call last):
  File "/home/dock/catkin_ws/src/src/gazebo_pkg_tests/slam_example/src/scripts/depth_to_laser.py", line 5, in <module>
    from depth_to_laser import DepthImageToLaserScan
  File "/home/dock/workspace/src/gazebo_pkg_tests/slam_example/src/scripts/depth_to_laser.py", line 5, in <module>
    from depth_to_laser import DepthImageToLaserScan
ImportError: cannot import name DepthImageToLaserScan
[1m[7m%[27m[1m[0m                                                                                                                                                            ]2;dock@reds-HP-Laptop:~/workspace/src/gazebo_pkg_tests/slam_example/src/scripts]1;..e/src/scripts[0m[27m[24m[J[0m[49m[39m[0m[49m[38;5;208mÓÇ∂[0m[48;5;208m[37m Ôåõ [0m[48;5;208m[37m [0m[40m[38;5;208mÓÇ∞[0m[40m[33m dock[0m[40m[33m [0m[44m[30mÓÇ∞[0m[44m[30m ÔÅº  scripts[0m[44m[30m [0m[48;5;28m[34mÓÇ∞[0m[48;5;28m[38;5;220m Óúº  2.7.12 (py27)[0m[48;5;28m[38;5;220m [0m[43m[38;5;28mÓÇ∞[0m[43m[30m ÔÑì  [0m[43m[30mÔÑ¶ main[0m[43m[30m[0m[43m[30m ÔÅ™ [0m[43m[30m ÔÅô [0m[43m[30m [0m[40m[33mÓÇ∞[0m[40m[31m ÔÄç[0m[40m[31m [0m[49m[30mÓÇ∞[0m[49m[39m [0m[49m[39m[K[?1h=[?2004h[32mrosrun[39m slam_example [35mdepth_to_laser.py[39m[?1l>[?2004l
]2;rosrun slam_example depth_to_laser.py]1;rosrunlibGL error: pci id for fd 33: 8086:8a52, driver (null)
libGL error: No driver found
libGL error: failed to load driver: (null)
libGL error: pci id for fd 33: 8086:8a52, driver (null)
libGL error: unable to load driver: iris_dri.so
libGL error: driver pointer missing
libGL error: failed to load driver: iris
^CUnhandled exception in thread started by 
sys.excepthook is missing
lost sys.stderr
QObject::~QObject: Timers cannot be stopped from another thread
[1m[7m%[27m[1m[0m                                                                                                                                                            ]2;dock@reds-HP-Laptop:~/workspace/src/gazebo_pkg_tests/slam_example/src/scripts]1;..e/src/scripts[0m[27m[24m[J[0m[49m[39m[0m[49m[38;5;208mÓÇ∂[0m[48;5;208m[37m Ôåõ [0m[48;5;208m[37m [0m[40m[38;5;208mÓÇ∞[0m[40m[33m dock[0m[40m[33m [0m[44m[30mÓÇ∞[0m[44m[30m ÔÅº  scripts[0m[44m[30m [0m[48;5;28m[34mÓÇ∞[0m[48;5;28m[38;5;220m Óúº  2.7.12 (py27)[0m[48;5;28m[38;5;220m [0m[43m[38;5;28mÓÇ∞[0m[43m[30m ÔÑì  [0m[43m[30mÔÑ¶ main[0m[43m[30m[0m[43m[30m ÔÅ™ [0m[43m[30m ÔÅô [0m[43m[30m [0m[49m[33mÓÇ∞[0m[49m[39m [0m[49m[39m[K[?1h=[?2004h[32mrosrun[39m slam_example [35mdepth_to_laser.py[39m[?1l>[?2004l
]2;rosrun slam_example depth_to_laser.py]1;rosrun320
libGL error: pci id for fd 33: 8086:8a52, driver (null)
libGL error: No driver found
libGL error: failed to load driver: (null)
libGL error: pci id for fd 33: 8086:8a52, driver (null)
libGL error: unable to load driver: iris_dri.so
libGL error: driver pointer missing
libGL error: failed to load driver: iris
320
320
320
320
320
320
320
320
320
320
320
320
320
320
320
320
320
320
320
320
320
320
^CQObject::~QObject: Timers cannot be stopped from another thread
^C
[1m[7m%[27m[1m[0m                                                                                                                                                            ]2;dock@reds-HP-Laptop:~/workspace/src/gazebo_pkg_tests/slam_example/src/scripts]1;..e/src/scripts[0m[27m[24m[J[0m[49m[39m[0m[49m[38;5;208mÓÇ∂[0m[48;5;208m[37m Ôåõ [0m[48;5;208m[37m [0m[40m[38;5;208mÓÇ∞[0m[40m[33m dock[0m[40m[33m [0m[44m[30mÓÇ∞[0m[44m[30m ÔÅº  scripts[0m[44m[30m [0m[48;5;28m[34mÓÇ∞[0m[48;5;28m[38;5;220m Óúº  2.7.12 (py27)[0m[48;5;28m[38;5;220m [0m[43m[38;5;28mÓÇ∞[0m[43m[30m ÔÑì  [0m[43m[30mÔÑ¶ main[0m[43m[30m[0m[43m[30m ÔÅ™ [0m[43m[30m ÔÅô [0m[43m[30m [0m[40m[33mÓÇ∞[0m[40m[31m ÔÄç[0m[40m[31m [0m[49m[30mÓÇ∞[0m[49m[39m [0m[49m[39m[K[?1h=[?2004h[?2004l
[1m[7m%[27m[1m[0m                                                                                                                                                            ]2;dock@reds-HP-Laptop:~/workspace/src/gazebo_pkg_tests/slam_example/src/scripts]1;..e/src/scripts[0m[27m[24m[J[0m[49m[39m[0m[49m[38;5;208mÓÇ∂[0m[48;5;208m[37m Ôåõ [0m[48;5;208m[37m [0m[40m[38;5;208mÓÇ∞[0m[40m[33m dock[0m[40m[33m [0m[44m[30mÓÇ∞[0m[44m[30m ÔÅº  scripts[0m[44m[30m [0m[48;5;28m[34mÓÇ∞[0m[48;5;28m[38;5;220m Óúº  2.7.12 (py27)[0m[48;5;28m[38;5;220m [0m[43m[38;5;28mÓÇ∞[0m[43m[30m ÔÑì  [0m[43m[30mÔÑ¶ main[0m[43m[30m[0m[43m[30m ÔÅ™ [0m[43m[30m ÔÅô [0m[43m[30m [0m[40m[33mÓÇ∞[0m[40m[31m ÔÄç[0m[40m[31m [0m[49m[30mÓÇ∞[0m[49m[39m [0m[49m[39m[K[?1h=[?2004h[32mrosrun[39m slam_example [35mdepth_to_laser.py[39m[?1l>[?2004l
]2;rosrun slam_example depth_to_laser.py]1;rosrun320
[31m[ERROR] [1680598723.100876, 854.068000]: bad callback: <bound method DepthToLaserConverter.depth_callback of <__main__.DepthToLaserConverter instance at 0x7fef2041f200>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/dock/catkin_ws/src/src/gazebo_pkg_tests/slam_example/src/scripts/depth_to_laser.py", line 31, in depth_callback
    scan_angles_degrees=list(np.linspace(-120, 120, 45)), should_update_map=True)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 259, in update
    CoreSLAM.update(self, scans_mm, pose_change, scan_angles_degrees, should_update_map)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 106, in update
    self._scan_update(self.scan_for_mapbuild, scans_mm, velocities, scan_angles_degrees)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 140, in _scan_update
    velocities=velocities, scan_angles_degrees=scan_angles_degrees)
TypeError: Wrong/insufficient arguments passed to Scan.update(): number of scan angles must equal number of scan distances

[0m320
[31m[ERROR] [1680598723.179553, 854.116000]: bad callback: <bound method DepthToLaserConverter.depth_callback of <__main__.DepthToLaserConverter instance at 0x7fef2041f200>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/dock/catkin_ws/src/src/gazebo_pkg_tests/slam_example/src/scripts/depth_to_laser.py", line 31, in depth_callback
    scan_angles_degrees=list(np.linspace(-120, 120, 45)), should_update_map=True)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 259, in update
    CoreSLAM.update(self, scans_mm, pose_change, scan_angles_degrees, should_update_map)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 106, in update
    self._scan_update(self.scan_for_mapbuild, scans_mm, velocities, scan_angles_degrees)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 140, in _scan_update
    velocities=velocities, scan_angles_degrees=scan_angles_degrees)
TypeError: Wrong/insufficient arguments passed to Scan.update(): number of scan angles must equal number of scan distances

[0m320
[31m[ERROR] [1680598723.268802, 854.164000]: bad callback: <bound method DepthToLaserConverter.depth_callback of <__main__.DepthToLaserConverter instance at 0x7fef2041f200>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/dock/catkin_ws/src/src/gazebo_pkg_tests/slam_example/src/scripts/depth_to_laser.py", line 31, in depth_callback
    scan_angles_degrees=list(np.linspace(-120, 120, 45)), should_update_map=True)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 259, in update
    CoreSLAM.update(self, scans_mm, pose_change, scan_angles_degrees, should_update_map)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 106, in update
    self._scan_update(self.scan_for_mapbuild, scans_mm, velocities, scan_angles_degrees)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 140, in _scan_update
    velocities=velocities, scan_angles_degrees=scan_angles_degrees)
TypeError: Wrong/insufficient arguments passed to Scan.update(): number of scan angles must equal number of scan distances

[0m320
[31m[ERROR] [1680598723.366565, 854.208000]: bad callback: <bound method DepthToLaserConverter.depth_callback of <__main__.DepthToLaserConverter instance at 0x7fef2041f200>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/dock/catkin_ws/src/src/gazebo_pkg_tests/slam_example/src/scripts/depth_to_laser.py", line 31, in depth_callback
    scan_angles_degrees=list(np.linspace(-120, 120, 45)), should_update_map=True)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 259, in update
    CoreSLAM.update(self, scans_mm, pose_change, scan_angles_degrees, should_update_map)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 106, in update
    self._scan_update(self.scan_for_mapbuild, scans_mm, velocities, scan_angles_degrees)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 140, in _scan_update
    velocities=velocities, scan_angles_degrees=scan_angles_degrees)
TypeError: Wrong/insufficient arguments passed to Scan.update(): number of scan angles must equal number of scan distances

[0m320
[31m[ERROR] [1680598723.438403, 854.247000]: bad callback: <bound method DepthToLaserConverter.depth_callback of <__main__.DepthToLaserConverter instance at 0x7fef2041f200>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/dock/catkin_ws/src/src/gazebo_pkg_tests/slam_example/src/scripts/depth_to_laser.py", line 31, in depth_callback
    scan_angles_degrees=list(np.linspace(-120, 120, 45)), should_update_map=True)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 259, in update
    CoreSLAM.update(self, scans_mm, pose_change, scan_angles_degrees, should_update_map)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 106, in update
    self._scan_update(self.scan_for_mapbuild, scans_mm, velocities, scan_angles_degrees)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 140, in _scan_update
    velocities=velocities, scan_angles_degrees=scan_angles_degrees)
TypeError: Wrong/insufficient arguments passed to Scan.update(): number of scan angles must equal number of scan distances

[0m320
[31m[ERROR] [1680598723.518312, 854.289000]: bad callback: <bound method DepthToLaserConverter.depth_callback of <__main__.DepthToLaserConverter instance at 0x7fef2041f200>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/dock/catkin_ws/src/src/gazebo_pkg_tests/slam_example/src/scripts/depth_to_laser.py", line 31, in depth_callback
    scan_angles_degrees=list(np.linspace(-120, 120, 45)), should_update_map=True)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 259, in update
    CoreSLAM.update(self, scans_mm, pose_change, scan_angles_degrees, should_update_map)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 106, in update
    self._scan_update(self.scan_for_mapbuild, scans_mm, velocities, scan_angles_degrees)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 140, in _scan_update
    velocities=velocities, scan_angles_degrees=scan_angles_degrees)
TypeError: Wrong/insufficient arguments passed to Scan.update(): number of scan angles must equal number of scan distances

[0m320
[31m[ERROR] [1680598723.658769, 854.342000]: bad callback: <bound method DepthToLaserConverter.depth_callback of <__main__.DepthToLaserConverter instance at 0x7fef2041f200>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/dock/catkin_ws/src/src/gazebo_pkg_tests/slam_example/src/scripts/depth_to_laser.py", line 31, in depth_callback
    scan_angles_degrees=list(np.linspace(-120, 120, 45)), should_update_map=True)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 259, in update
    CoreSLAM.update(self, scans_mm, pose_change, scan_angles_degrees, should_update_map)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 106, in update
    self._scan_update(self.scan_for_mapbuild, scans_mm, velocities, scan_angles_degrees)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 140, in _scan_update
    velocities=velocities, scan_angles_degrees=scan_angles_degrees)
TypeError: Wrong/insufficient arguments passed to Scan.update(): number of scan angles must equal number of scan distances

[0m320
[31m[ERROR] [1680598723.740361, 854.383000]: bad callback: <bound method DepthToLaserConverter.depth_callback of <__main__.DepthToLaserConverter instance at 0x7fef2041f200>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/dock/catkin_ws/src/src/gazebo_pkg_tests/slam_example/src/scripts/depth_to_laser.py", line 31, in depth_callback
    scan_angles_degrees=list(np.linspace(-120, 120, 45)), should_update_map=True)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 259, in update
    CoreSLAM.update(self, scans_mm, pose_change, scan_angles_degrees, should_update_map)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 106, in update
    self._scan_update(self.scan_for_mapbuild, scans_mm, velocities, scan_angles_degrees)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 140, in _scan_update
    velocities=velocities, scan_angles_degrees=scan_angles_degrees)
TypeError: Wrong/insufficient arguments passed to Scan.update(): number of scan angles must equal number of scan distances

[0m320
[31m[ERROR] [1680598723.834513, 854.432000]: bad callback: <bound method DepthToLaserConverter.depth_callback of <__main__.DepthToLaserConverter instance at 0x7fef2041f200>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/dock/catkin_ws/src/src/gazebo_pkg_tests/slam_example/src/scripts/depth_to_laser.py", line 31, in depth_callback
    scan_angles_degrees=list(np.linspace(-120, 120, 45)), should_update_map=True)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 259, in update
    CoreSLAM.update(self, scans_mm, pose_change, scan_angles_degrees, should_update_map)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 106, in update
    self._scan_update(self.scan_for_mapbuild, scans_mm, velocities, scan_angles_degrees)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 140, in _scan_update
    velocities=velocities, scan_angles_degrees=scan_angles_degrees)
TypeError: Wrong/insufficient arguments passed to Scan.update(): number of scan angles must equal number of scan distances

[0m320
[31m[ERROR] [1680598723.969153, 854.494000]: bad callback: <bound method DepthToLaserConverter.depth_callback of <__main__.DepthToLaserConverter instance at 0x7fef2041f200>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/dock/catkin_ws/src/src/gazebo_pkg_tests/slam_example/src/scripts/depth_to_laser.py", line 31, in depth_callback
    scan_angles_degrees=list(np.linspace(-120, 120, 45)), should_update_map=True)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 259, in update
    CoreSLAM.update(self, scans_mm, pose_change, scan_angles_degrees, should_update_map)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 106, in update
    self._scan_update(self.scan_for_mapbuild, scans_mm, velocities, scan_angles_degrees)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 140, in _scan_update
    velocities=velocities, scan_angles_degrees=scan_angles_degrees)
TypeError: Wrong/insufficient arguments passed to Scan.update(): number of scan angles must equal number of scan distances

[0m320
[31m[ERROR] [1680598724.083170, 854.528000]: bad callback: <bound method DepthToLaserConverter.depth_callback of <__main__.DepthToLaserConverter instance at 0x7fef2041f200>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/dock/catkin_ws/src/src/gazebo_pkg_tests/slam_example/src/scripts/depth_to_laser.py", line 31, in depth_callback
    scan_angles_degrees=list(np.linspace(-120, 120, 45)), should_update_map=True)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 259, in update
    CoreSLAM.update(self, scans_mm, pose_change, scan_angles_degrees, should_update_map)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 106, in update
    self._scan_update(self.scan_for_mapbuild, scans_mm, velocities, scan_angles_degrees)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 140, in _scan_update
    velocities=velocities, scan_angles_degrees=scan_angles_degrees)
TypeError: Wrong/insufficient arguments passed to Scan.update(): number of scan angles must equal number of scan distances

[0m320
[31m[ERROR] [1680598724.176691, 854.583000]: bad callback: <bound method DepthToLaserConverter.depth_callback of <__main__.DepthToLaserConverter instance at 0x7fef2041f200>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/dock/catkin_ws/src/src/gazebo_pkg_tests/slam_example/src/scripts/depth_to_laser.py", line 31, in depth_callback
    scan_angles_degrees=list(np.linspace(-120, 120, 45)), should_update_map=True)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 259, in update
    CoreSLAM.update(self, scans_mm, pose_change, scan_angles_degrees, should_update_map)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 106, in update
    self._scan_update(self.scan_for_mapbuild, scans_mm, velocities, scan_angles_degrees)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 140, in _scan_update
    velocities=velocities, scan_angles_degrees=scan_angles_degrees)
TypeError: Wrong/insufficient arguments passed to Scan.update(): number of scan angles must equal number of scan distances

[0m320
[31m[ERROR] [1680598724.262934, 854.624000]: bad callback: <bound method DepthToLaserConverter.depth_callback of <__main__.DepthToLaserConverter instance at 0x7fef2041f200>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/dock/catkin_ws/src/src/gazebo_pkg_tests/slam_example/src/scripts/depth_to_laser.py", line 31, in depth_callback
    scan_angles_degrees=list(np.linspace(-120, 120, 45)), should_update_map=True)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 259, in update
    CoreSLAM.update(self, scans_mm, pose_change, scan_angles_degrees, should_update_map)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 106, in update
    self._scan_update(self.scan_for_mapbuild, scans_mm, velocities, scan_angles_degrees)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 140, in _scan_update
    velocities=velocities, scan_angles_degrees=scan_angles_degrees)
TypeError: Wrong/insufficient arguments passed to Scan.update(): number of scan angles must equal number of scan distances

[0m320
[31m[ERROR] [1680598724.387585, 854.664000]: bad callback: <bound method DepthToLaserConverter.depth_callback of <__main__.DepthToLaserConverter instance at 0x7fef2041f200>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/dock/catkin_ws/src/src/gazebo_pkg_tests/slam_example/src/scripts/depth_to_laser.py", line 31, in depth_callback
    scan_angles_degrees=list(np.linspace(-120, 120, 45)), should_update_map=True)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 259, in update
    CoreSLAM.update(self, scans_mm, pose_change, scan_angles_degrees, should_update_map)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 106, in update
    self._scan_update(self.scan_for_mapbuild, scans_mm, velocities, scan_angles_degrees)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 140, in _scan_update
    velocities=velocities, scan_angles_degrees=scan_angles_degrees)
TypeError: Wrong/insufficient arguments passed to Scan.update(): number of scan angles must equal number of scan distances

[0m320
[31m[ERROR] [1680598724.484038, 854.710000]: bad callback: <bound method DepthToLaserConverter.depth_callback of <__main__.DepthToLaserConverter instance at 0x7fef2041f200>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/dock/catkin_ws/src/src/gazebo_pkg_tests/slam_example/src/scripts/depth_to_laser.py", line 31, in depth_callback
    scan_angles_degrees=list(np.linspace(-120, 120, 45)), should_update_map=True)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 259, in update
    CoreSLAM.update(self, scans_mm, pose_change, scan_angles_degrees, should_update_map)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 106, in update
    self._scan_update(self.scan_for_mapbuild, scans_mm, velocities, scan_angles_degrees)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 140, in _scan_update
    velocities=velocities, scan_angles_degrees=scan_angles_degrees)
TypeError: Wrong/insufficient arguments passed to Scan.update(): number of scan angles must equal number of scan distances

[0m320
[31m[ERROR] [1680598724.553473, 854.755000]: bad callback: <bound method DepthToLaserConverter.depth_callback of <__main__.DepthToLaserConverter instance at 0x7fef2041f200>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/dock/catkin_ws/src/src/gazebo_pkg_tests/slam_example/src/scripts/depth_to_laser.py", line 31, in depth_callback
    scan_angles_degrees=list(np.linspace(-120, 120, 45)), should_update_map=True)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 259, in update
    CoreSLAM.update(self, scans_mm, pose_change, scan_angles_degrees, should_update_map)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 106, in update
    self._scan_update(self.scan_for_mapbuild, scans_mm, velocities, scan_angles_degrees)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 140, in _scan_update
    velocities=velocities, scan_angles_degrees=scan_angles_degrees)
TypeError: Wrong/insufficient arguments passed to Scan.update(): number of scan angles must equal number of scan distances

[0m320
[31m[ERROR] [1680598724.678115, 854.826000]: bad callback: <bound method DepthToLaserConverter.depth_callback of <__main__.DepthToLaserConverter instance at 0x7fef2041f200>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/dock/catkin_ws/src/src/gazebo_pkg_tests/slam_example/src/scripts/depth_to_laser.py", line 31, in depth_callback
    scan_angles_degrees=list(np.linspace(-120, 120, 45)), should_update_map=True)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 259, in update
    CoreSLAM.update(self, scans_mm, pose_change, scan_angles_degrees, should_update_map)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 106, in update
    self._scan_update(self.scan_for_mapbuild, scans_mm, velocities, scan_angles_degrees)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 140, in _scan_update
    velocities=velocities, scan_angles_degrees=scan_angles_degrees)
TypeError: Wrong/insufficient arguments passed to Scan.update(): number of scan angles must equal number of scan distances

[0m320
[31m[ERROR] [1680598724.801917, 854.859000]: bad callback: <bound method DepthToLaserConverter.depth_callback of <__main__.DepthToLaserConverter instance at 0x7fef2041f200>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/dock/catkin_ws/src/src/gazebo_pkg_tests/slam_example/src/scripts/depth_to_laser.py", line 31, in depth_callback
    scan_angles_degrees=list(np.linspace(-120, 120, 45)), should_update_map=True)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 259, in update
    CoreSLAM.update(self, scans_mm, pose_change, scan_angles_degrees, should_update_map)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 106, in update
    self._scan_update(self.scan_for_mapbuild, scans_mm, velocities, scan_angles_degrees)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 140, in _scan_update
    velocities=velocities, scan_angles_degrees=scan_angles_degrees)
TypeError: Wrong/insufficient arguments passed to Scan.update(): number of scan angles must equal number of scan distances

[0m320
[31m[ERROR] [1680598724.902613, 854.916000]: bad callback: <bound method DepthToLaserConverter.depth_callback of <__main__.DepthToLaserConverter instance at 0x7fef2041f200>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/dock/catkin_ws/src/src/gazebo_pkg_tests/slam_example/src/scripts/depth_to_laser.py", line 31, in depth_callback
    scan_angles_degrees=list(np.linspace(-120, 120, 45)), should_update_map=True)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 259, in update
    CoreSLAM.update(self, scans_mm, pose_change, scan_angles_degrees, should_update_map)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 106, in update
    self._scan_update(self.scan_for_mapbuild, scans_mm, velocities, scan_angles_degrees)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 140, in _scan_update
    velocities=velocities, scan_angles_degrees=scan_angles_degrees)
TypeError: Wrong/insufficient arguments passed to Scan.update(): number of scan angles must equal number of scan distances

[0m320
[31m[ERROR] [1680598724.999204, 854.954000]: bad callback: <bound method DepthToLaserConverter.depth_callback of <__main__.DepthToLaserConverter instance at 0x7fef2041f200>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/dock/catkin_ws/src/src/gazebo_pkg_tests/slam_example/src/scripts/depth_to_laser.py", line 31, in depth_callback
    scan_angles_degrees=list(np.linspace(-120, 120, 45)), should_update_map=True)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 259, in update
    CoreSLAM.update(self, scans_mm, pose_change, scan_angles_degrees, should_update_map)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 106, in update
    self._scan_update(self.scan_for_mapbuild, scans_mm, velocities, scan_angles_degrees)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 140, in _scan_update
    velocities=velocities, scan_angles_degrees=scan_angles_degrees)
TypeError: Wrong/insufficient arguments passed to Scan.update(): number of scan angles must equal number of scan distances

[0m320
[31m[ERROR] [1680598725.112771, 855.001000]: bad callback: <bound method DepthToLaserConverter.depth_callback of <__main__.DepthToLaserConverter instance at 0x7fef2041f200>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/dock/catkin_ws/src/src/gazebo_pkg_tests/slam_example/src/scripts/depth_to_laser.py", line 31, in depth_callback
    scan_angles_degrees=list(np.linspace(-120, 120, 45)), should_update_map=True)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 259, in update
    CoreSLAM.update(self, scans_mm, pose_change, scan_angles_degrees, should_update_map)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 106, in update
    self._scan_update(self.scan_for_mapbuild, scans_mm, velocities, scan_angles_degrees)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 140, in _scan_update
    velocities=velocities, scan_angles_degrees=scan_angles_degrees)
TypeError: Wrong/insufficient arguments passed to Scan.update(): number of scan angles must equal number of scan distances

[0m320
[31m[ERROR] [1680598725.198244, 855.030000]: bad callback: <bound method DepthToLaserConverter.depth_callback of <__main__.DepthToLaserConverter instance at 0x7fef2041f200>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/dock/catkin_ws/src/src/gazebo_pkg_tests/slam_example/src/scripts/depth_to_laser.py", line 31, in depth_callback
    scan_angles_degrees=list(np.linspace(-120, 120, 45)), should_update_map=True)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 259, in update
    CoreSLAM.update(self, scans_mm, pose_change, scan_angles_degrees, should_update_map)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 106, in update
    self._scan_update(self.scan_for_mapbuild, scans_mm, velocities, scan_angles_degrees)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 140, in _scan_update
    velocities=velocities, scan_angles_degrees=scan_angles_degrees)
TypeError: Wrong/insufficient arguments passed to Scan.update(): number of scan angles must equal number of scan distances

[0m320
[31m[ERROR] [1680598725.277818, 855.067000]: bad callback: <bound method DepthToLaserConverter.depth_callback of <__main__.DepthToLaserConverter instance at 0x7fef2041f200>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/dock/catkin_ws/src/src/gazebo_pkg_tests/slam_example/src/scripts/depth_to_laser.py", line 31, in depth_callback
    scan_angles_degrees=list(np.linspace(-120, 120, 45)), should_update_map=True)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 259, in update
    CoreSLAM.update(self, scans_mm, pose_change, scan_angles_degrees, should_update_map)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 106, in update
    self._scan_update(self.scan_for_mapbuild, scans_mm, velocities, scan_angles_degrees)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 140, in _scan_update
    velocities=velocities, scan_angles_degrees=scan_angles_degrees)
TypeError: Wrong/insufficient arguments passed to Scan.update(): number of scan angles must equal number of scan distances

[0m320
[31m[ERROR] [1680598725.355130, 855.107000]: bad callback: <bound method DepthToLaserConverter.depth_callback of <__main__.DepthToLaserConverter instance at 0x7fef2041f200>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/dock/catkin_ws/src/src/gazebo_pkg_tests/slam_example/src/scripts/depth_to_laser.py", line 31, in depth_callback
    scan_angles_degrees=list(np.linspace(-120, 120, 45)), should_update_map=True)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 259, in update
    CoreSLAM.update(self, scans_mm, pose_change, scan_angles_degrees, should_update_map)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 106, in update
    self._scan_update(self.scan_for_mapbuild, scans_mm, velocities, scan_angles_degrees)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 140, in _scan_update
    velocities=velocities, scan_angles_degrees=scan_angles_degrees)
TypeError: Wrong/insufficient arguments passed to Scan.update(): number of scan angles must equal number of scan distances

[0m320
[31m[ERROR] [1680598725.510879, 855.183000]: bad callback: <bound method DepthToLaserConverter.depth_callback of <__main__.DepthToLaserConverter instance at 0x7fef2041f200>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/dock/catkin_ws/src/src/gazebo_pkg_tests/slam_example/src/scripts/depth_to_laser.py", line 31, in depth_callback
    scan_angles_degrees=list(np.linspace(-120, 120, 45)), should_update_map=True)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 259, in update
    CoreSLAM.update(self, scans_mm, pose_change, scan_angles_degrees, should_update_map)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 106, in update
    self._scan_update(self.scan_for_mapbuild, scans_mm, velocities, scan_angles_degrees)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 140, in _scan_update
    velocities=velocities, scan_angles_degrees=scan_angles_degrees)
TypeError: Wrong/insufficient arguments passed to Scan.update(): number of scan angles must equal number of scan distances

[0m320
[31m[ERROR] [1680598725.607741, 855.243000]: bad callback: <bound method DepthToLaserConverter.depth_callback of <__main__.DepthToLaserConverter instance at 0x7fef2041f200>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/dock/catkin_ws/src/src/gazebo_pkg_tests/slam_example/src/scripts/depth_to_laser.py", line 31, in depth_callback
    scan_angles_degrees=list(np.linspace(-120, 120, 45)), should_update_map=True)
  File "/usr/local/lib/python2.7/dist-packages/breezyslam/algorithms.py", line 259, in update
    CoreSLAM.update(self, scans_mm, pose_change, scan_angles_degrees, should_update_map)
  File "/usr/local/lib/python2.7/di